ghga_connector/core/httpyexpect_translation.py

```diff
@@ -0,0 +1,133 @@
+# Copyright 2021 - 2022 Universität Tübingen, DKFZ and EMBL
```

Comment:
Maybe rename this module to "httperror_translation.py" or simply "http_translation.py"